amulet/level/formats/anvil_world/format.py

```diff
@@ -321,6 +321,12 @@ def get_recursive(obj: nbt.TAG_Compound, *keys):
 
             self._bounds[dimension_name] = bounds
 
+    def _get_interface(self, raw_chunk_data: Optional[Any] = None) -> "Interface":
+        from amulet.level.loader import Interfaces
```

Comment:
Is there a reason why this import has to be in the function instead of at the top of the file?

Reply:
IIRC there was a circular import error that I should try and fix.
This was copied from the old code where I did the same it has just been moved.